assets/templates/partials/release/contents/about-the-data.tmpl

```diff
@@ -16,10 +16,7 @@
         alt="Welsh Government logo"
         class="ons-u-mb-xs ons-u-d-b"
       >
-      <p>
-        This release has been written with contributions of analysis and data
-        supplied by the Welsh Government.
-      </p>
+      <p>{{- localise "CensusPhrasingWelshStatistic" $.Language 1 -}}</p>
```

Comment:
This choice of key `CensusPhrasingWelshStatistic` risks confusing the welsh statistic designation with the census. I'd suggest replacing it with `AboutDataWelshStatistic` instead.